rules/detekt/src/test/kotlin/com/twitter/compose/rules/detekt/ComposePreviewPublicCheckTest.kt

```diff
@@ -56,4 +56,18 @@ class ComposePreviewPublicCheckTest {
             assertThat(error).hasMessage(ComposePreviewPublic.ComposablesPreviewShouldNotBePublic)
         }
     }
+
+    @Test
```

Comment:
you'll need to add this same test to `ComposePreviewPublicCheckTest` in the ktlint package